Tasks/GitHubReleaseV0/Tests/TestStrings.ts

```diff
@@ -0,0 +1,24 @@
+
+export class TestString {
+    public static readonly createReleaseSuccessKeyword = "CreateReleaseSuccess";
```

Comment:
Inconsistency - Few variables are defined with type while few are not.

Reply:
Will take in next PR